00-Getting_Started/get_inspektor_gadget.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Check that kubectl is installed
+if ! kubectl version >/dev/null 2>&1; then
+  echo "ERROR: Can't execute kubectl."
+  echo "Make sure kubectl is installed before installing Inspektor Gadget"
+  exit 1
+fi
+
+# Check if kubectl gadget already works
+if kubectl gadget version >/dev/null 2>&1; then
```

Comment:
```suggestion
if [ "$(kubectl gadget version)" != "v0.2.0" ] ; then```

It could check the correct version.